packages/components/select/src/select.vue

```diff
@@ -141,7 +141,6 @@
               </el-tooltip>
             </slot>
             <div
-              v-if="!selectDisabled"
```

Comment:
This judgment was specially added after the 2.5.0 refactoring. Do you still remember the reason why this judgment was added? @kooriookami